src/dropbox.js

```diff
@@ -354,14 +354,14 @@
 
     /**
      * Checks for the path in ``_revCache`` and decides based on that if file
-     * has changed. Calls ``_getFolder`` is the path points to a folder.
+     * has changed. Calls ``_getFolder`` if the path points to a folder.
      *
      * Calls ``Dropbox.share`` afterwards to fill ``_itemRefs``.
      *
      * Compatible with ``WireClient.get``
      *
-     * Checks for the path in _revCache and decides based on that if file has
-     * changed. Calls _getFolder if the path points to a folder.
```

Comment:
Why delete the explanation of what the function does?

Reply:
It's not deleted, it was already above. The Git merge looks weird because I fixed a typo as well (s/is/if)